source/logHandler.py

```diff
@@ -288,6 +288,19 @@ class Formatter(logging.Formatter):
 	def formatException(self, ex):
 		return stripBasePathFromTracebackText(super(Formatter, self).formatException(ex))
 
+	def formatTime(self, record, datefmt=None):

```

Comment:
```suggestion
	def formatTime(self, record: logging.LogRecord, datefmt: Optional[str] = None) -> str:
```

I know that python doesn't even do this nor should this be used directly, but it would be helpful for developers if we have to change this further